hw/mcu/dialog/da1469x/include/mcu/da1469x_otp.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef __MCU_DA1469X_OTP_H_
+#define __MCU_DA1469X_OTP_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define OTPM_BASE 0x10080000UL
+#define OTPM_SIZE 0x4096
```

Comment:
This means splitting off this change into another repo at this point.  I was hoping to keep this self contained.

Reply:
So I ended up moving these to the driver file itself.  The rationale is that the mcu.h currently resides in another repo and since only the driver now uses the definition, let's leave it there.  Ideally this would have been part of the CMSIS files provided by the manufacturer, but it is not.

Reply:
this is common code for da1469x family, you just need to update `mcu.h` for `da14699` - 3rd party repos need to be adjusted to core, not the other way around...

Reply:
or maybe it does.  not sure why this didnt work as I did try this yesterday but it worked today.   It might have been because I had the target set to mist vs dk.